packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.tsx

```diff
@@ -90,9 +96,12 @@ export class GuxFormFieldColor {
       <GuxFormFieldContainer labelPosition={this.computedLabelPosition}>
         <GuxFormFieldLabel
```

Comment:
I think we should add a `indicatorMark` property to this component and keep the required and optional stuff together.

Reply:
Moved that stuff into the label component